proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -388,28 +393,28 @@ message ReceiveComponentDataStreamRequest {
     //     then the stream will return an error.
     //
     // !!! note
-    //     Components may not support all metrics. If a component does not
-    //     support a given metric, then the returned data stream will not
-    //     contain that metric.
-    repeated frequenz.api.common.v1.metrics.Metric metrics = 1;
+    //     Components may not support all metrics. If a electrical component
+    //     does not support a given metric, then the returned data stream will
+    //     not contain that metric.
+    repeated frequenz.api.common.v1alpha7.metrics.Metric metrics = 1;
   }
 
-  // The component ID to subscribe to.
-  uint64 component_id = 1;
+  // The electrical component ID to subscribe to.
+  uint64 electrical_component_id = 1;
 
   // The filter to apply to the stream.
   //
   // This field is optional. If this is not provided, then the stream will
-  // return all metrics for the given component. If this is provided, then the
-  // stream will return only the metrics specified in the filter.
+  // return all metrics for the given electrical component. If this is provided,
+  // then the stream will return only the metrics specified in the filter.
   ComponentDataStreamFilter filter = 2;
 }
 
-// A data sample from a component in the microgrid.
-// Components belonging to all categories and types can be represented using
-// this message.
-message ReceiveComponentDataStreamResponse {
-  frequenz.api.common.v1.microgrid.components.ComponentData data = 1;
+// A data sample from a electrical component in the microgrid.
+// Electrical Components belonging to all categories and types can be
+// represented using this message.
+message ReceiveElectricalComponentDataStreamResponse {
+  frequenz.api.common.v1alpha7.microgrid.electrical_components.ElectricalComponentTelemetry data = 1;
```

Comment:
Also wonder if we want to change this to `telemetry` to make it consistent with the message name.